lib/Db/TagMapper.php

```diff
@@ -145,20 +145,22 @@ public function getAllTagsForMessages(array $messages): array {
 
 		$qb = $this->db->getQueryBuilder();
 		$idsQuery = $qb->select('mt.*')
+			->addSelect('t.*')
 			->from('mail_message_tags', 'mt')
+			->join('mt', $this->getTableName(), 't', $qb->expr()->eq('mt.tag_id','t.id'))
```

Comment:
for oracle compat we should always specify the `eq` type parameter when the comparison doesn't compare strings

Reply:
done